lua/lsp_extensions/dart/closing_labels.lua

```diff
@@ -0,0 +1,69 @@
+--[[
+## Closing Labels
+
+** Method: 'dart/textDocument/publishClosingLabels**
+Direction: Server -> Client Params: { uri: string, labels: { label: string, range: Range }[] }
+This notifies the client when closing label information is available (or updated) for a file.
+
+Since this is a notification, the callback needs to be registered in the client's callbacks table.
+This can be achieved with nvim_lspconfig with this minimal config.
+```lua
+nvim_lsp.dartls.setup{
+  init_options = {
+    closingLabels = true,
+  },
+  callbacks = {
+    ['dart/textDocument/publishClosingLabels'] = require('lsp_extensions.dart.closing_labels').get_callback{},
```

Comment:
Hmmm is there any way to do this not in the setup, but some time afterward? This way we could do it for the user.

Reply:
I think we  can just set vim.lsp.callbacks['dart/...'] equal to the callback we want.

Reply:
A response to this is included here
https://github.com/tjdevries/lsp_extensions.nvim/pull/6#discussion_r495537495